### ISSUE TITLE
CoC support

### ISSUE THREAD

**Issue by revmischa** [toricls/cobolambda#1]
Needs [cobol on cogs](http://www.coboloncogs.org/INDEX.HTM) support

**Comment by toricls** [comment]
@revmischa I totally agree with you. Let's take a look at [cobol-on-wheelchair](https://github.com/azac/cobol-on-wheelchair) to get started 💪
